source/_integrations/nfandroidtv.markdown

```diff
@@ -125,4 +125,20 @@ icon:
   auth: "digest"
 ```
 
+Example of automation serivce call, full cofiguration:
+
+```yaml
+  - service: notify.living_room_tv
+    data:
+      title: "Thanks for the water!"
+      message: "Nigel is {{ states.sensor.nigel_moisture.state }}% moisture"
+      data:
+        duration: 4
+        position: "bottom-left"
+        fontsize: "medium"
+        transparency: "80%"
+        color: "teal"
+        interrupt: 0
+ ```yaml
```

Comment:
You can remove `yaml` here

Reply:
I'm not sure how this works 'you can remove yaml here' - I am clicking on th etext but I can't edit ithere.  I'll go back to the PR and try.  Thanks again for your patience.